sys/netinet/in_mcast.c

```diff
@@ -485,7 +485,7 @@ imo_multi_filter(const struct ip_moptions *imo, const struct ifnet *ifp,
 	ims = imo_match_source(imf, src);
 
 	if ((ims == NULL && mode == MCAST_INCLUDE) ||
-	    (ims != NULL && ims->imsl_st[0] != mode))
+		(ims != NULL && ims->imsl_st[0] == MCAST_EXCLUDE ))
```

Comment:
```suggestion
	    (ims != NULL && ims->imsl_st[0] == MCAST_EXCLUDE))
```

Reply:
already did these in my test tree. It's so trivial there's no need to ask for this to be redone.

Reply:
Fair :)! I figured it was worth showing the proposed changes, since it would help the contributor understand our style(9) norms/standards :).